lms/djangoapps/shoppingcart/views.py

```diff
@@ -595,6 +596,35 @@ def donate(request):
     return HttpResponse(response_params, content_type="text/json")
 
 
+def _get_verify_flow_redirect(order):
+    """Check if we're in the verification flow and redirect if necessary.
+
+    Arguments:
+        order (Order): The order received by the post-pay callback.
+
+    Returns:
+        HttpResponseRedirect or None
+
+    """
+    # See if the order contained any certificate items
+    # If so, the user is coming from the payment/verification flow.
+    cert_items = CertificateItem.objects.filter(order=order)
+
+    # Currently, we allow the purchase of only one verified
+    # enrollment at a time; if there are more than one,
+    # this will choose the first.
+    if cert_items.count() > 0:
```

Comment:
nitpick but is it worth having some logging in case there are many cases where there are more than a single verified enrollment?